docsite/rst/intro_configuration.rst

```diff
@@ -897,3 +897,19 @@ The normal behaviour is for operations to copy the existing context or use the u
 The default list is: nfs,vboxsf,fuse,ramfs::
 
     special_context_filesystems = nfs,vboxsf,fuse,ramfs,myspecialfs
+
+Galaxy Settings
+---------------
+
+The following options can be set in the [galaxy] section of ansible.cfg:
+
+server
+======
+
+Override the default Galaxy server value of https://galaxy.ansible.com.
```

Comment:
we should point out here, that this is only useful if you have a hosted version of the galaxy web app, this won't work against private hosted repos which galaxy can still use for fetching roles.